packages/cli/config-management/manage-generator/src/generatorInvocations.ts

```diff
@@ -2,23 +2,23 @@ import { GeneratorInvocationSchema, GeneratorName } from "@fern-api/generators-c
 
 export const GENERATOR_INVOCATIONS: Record<GeneratorName, Omit<GeneratorInvocationSchema, "name">> = {
     [GeneratorName.JAVA]: {
-        version: "0.0.122",
+        version: "0.3.7",
```

Comment:
this is a legacy generator, will want to keep it at 0.0.122

Reply:
roger that. reverted that change in a new commit.